dist/rpm/openQA.spec

```diff
@@ -314,6 +314,24 @@ Use this package to install munin scripts that allow to monitor some openQA
 statistics.
 %endif
 
+%package client-bash-completion
+Summary:        Bash Completion for %{name}
+Requires:       bash-completion
+Supplements:    (%{name}-client and bash)
+
+%description client-bash-completion
+The official bash completion script for openqa-cli.
+
+install -Dm 0644 completion/openqa-cli-completion.bash %{buildroot}%{_datadir}/bash-completion/completions/openqa-cli
+
+%package client-zsh-completion
+Summary:        Zsh Completion for %{name}
+Supplements:    (%{name}-client and zsh)
+
+%description client-zsh-completion
+The official zsh completion script for openqa-cli.
+
+install -Dm 0644 completion/openqa-cli-completion.zsh %{buildroot}%{_datadir}/usr/share/zsh/site-functions/_openqa-cli
```

Comment:
Now that string is part of the description :)

It needs to be in the `%install` section.

Reply:
I am working on that.